geom/gdml/src/TGDMLWrite.cxx

```diff
@@ -2026,7 +2026,7 @@ TString TGDMLWrite::GenName(TString oldname, TString objPointer)
          break;
       case kwithoutSufixNotUniq:
          //no change
-         newname = newname;
+         static_cast<void>(newname);
```

Comment:
`(void)newname` is sufficient - this is the common pattern for "and know this is unused". And do we need this at all? Can we simply remove this line?

Reply:
I removed it! Thanks!